component-test/test_dbus_proxy.py

```diff
@@ -486,6 +486,85 @@ def test_disallowed_iface_is_not_accessible(self,
         assert "My unique key" not in captured_stdout
 
 
+class TestProxyMethods(object):
+    """ This class tests methods configurations. There are two ways of setting methods
+        a string representing a method or an array of string representing couple of methods.
```

Comment:
Make it clear that it's "one string or an array of strings", rather than "a couple" which might be misunderstood.